profiler/memory.go

```diff
@@ -102,19 +102,18 @@ func NewMemoryWithShutdown(path string, interval time.Duration, shutdownC chan s
 func (memory *Memory) Run() {
 	log := pfxlog.Logger()
 	log.Infof("memory profiling to [%s]", memory.path)
```

Comment:
This should probably be 

    ticker := time.Ticker(memory.interval)
    defer ticker.Stop()

Reply:
It would actually be `time.NewTicker`, if you prefer an approach that includes a defer of the stopper.  I'll make that swap.